src/texttransform.js

```diff
@@ -30,13 +30,14 @@ function textTransform (text) {
     text = text.replace(/\nRequest URL:([^\n]|\n(?!\n))*/g, '```$&\n```  \n---');
 
     // Escape marked default handling of underscores, making it so
-    // underscores don't compile to italics
+    // underscores don't compile to bold

```

Comment:
In markdown `_text_` is turned into `<em>text</em>` or `<i>text</i>` (depending on the engine). Not `<strong>text</strong>` or `<b>text</b>`.